app/stac_api/serializers.py

```diff
@@ -156,10 +157,19 @@ class Meta:
     license = serializers.CharField(max_length=30)  # string
     links = CollectionLinkSerializer(many=True, read_only=True)
     providers = ProviderSerializer(many=True)
-    stac_extension = serializers.ListField(child=serializers.CharField(max_length=255),)
-    stac_version = serializers.CharField(max_length=10)  # string
+    stac_extensions = serializers.SerializerMethodField()
+    stac_version = serializers.SerializerMethodField()
     title = serializers.CharField(allow_blank=True, max_length=255)  # string
 
+    def get_crs(self, obj):
+        return ["http://www.opengis.net/def/crs/OGC/1.3/CRS84"]
+
+    def get_stac_extensions(self, obj):
+        return get_default_stac_extensions()
+
+    def get_stac_version(self, obj):
+        return "0.9.0"
```

Comment:
I would put the STAC version in a constant here, but this could also be done within https://jira.swisstopo.ch/browse/BGDIINF_SB-1402

Reply:
:+1: